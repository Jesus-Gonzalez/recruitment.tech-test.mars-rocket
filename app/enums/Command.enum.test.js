const Command = require('./Command.enum');

describe('enums', () => {
    describe('Command', () => {
        it('should return the Command object', () => {
            expect(Command).toMatchSnapshot();
        });
    });
});
