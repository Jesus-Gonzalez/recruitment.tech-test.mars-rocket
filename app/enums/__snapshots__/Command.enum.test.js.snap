// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`enums Command should return the Command object 1`] = `
Object {
  "Left": "L",
  "Move": "M",
  "Right": "R",
}
`;
